src/main/scala/com/hs/pipeline/demo/transformations/ContractTransformation.scala

```diff
@@ -14,4 +16,13 @@ object ContractTransformation {
     contractWithCountryCols
   }
 
+  def calParticipation(contractDf: DataFrame): DataFrame = {
+    val sumAmount = contractDf.groupBy("entity_id").agg(sum(contractDf("amount").cast(IntegerType)) as "sum_amount")
+    val interDf = contractDf.join(sumAmount,"entity_id")
+    val resultDf = interDf.withColumn("participation",
```

Comment:
avoid using hardcoded names, try to create a constant for the new column name and put it for example in BV schema file with all selected columns for the BV